mne_bids/mne_bids.py

```diff
@@ -499,43 +501,58 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     if len(events) > 0:
         _events_tsv(events, raw, events_tsv_fname, event_id, verbose)
 
-    # Write the neural data
-    # ---------------------
-    if os.path.exists(raw_file_bids) and not overwrite:
-        raise ValueError('"%s" already exists. Please set'
-                         ' overwrite to True.' % raw_file_bids)
-
-    if verbose:
-        print('Writing data files to %s' % raw_file_bids)
-
-    # Cover FIF
-    if ext in ['.fif', '.gz']:
+    # Writing of neural data: Different for MEG and EEG
+    # Check if it is MEG (only writing to FIF)
+    # ----------------------------------------
+    if ext in allowed_extensions_meg:
         # for FIF, we need to re-save the file to fix the file pointer
         # for files with multiple parts
-        raw.save(raw_file_bids, overwrite=overwrite)
-    # Cover CTF data
-    elif ext == '.ds':
-        # CTF data is saved in a directory
-        sh.copytree(raw_file, raw_file_bids)
-    # Cover BrainVision data
-    elif ext in ['.eeg', '.vhdr']:
-        # Multifile format with pointers within the single files
-        # Get future file names
-        fname, ext = _parse_ext(raw_file_bids)
-        raw_file_bids_list = [fname + '.eeg',
-                              fname + '.vhdr',
-                              fname + '.vmrk']
-        # Get locations of current files
-        fname, ext = _parse_ext(raw_file)
-        raw_file_list = [fname + '.eeg',
-                         fname + '.vhdr',
-                         fname + '.vmrk']
-
-        # Rewrite each file with adjusting the pointers
-        for raw_file, raw_file_bids in zip(raw_file_list, raw_file_bids_list):
-            copyfile_brainvision(raw_file, raw_file_bids)
-    # Cover all other data
+        if ext in ['.fif', '.gz']:
+            raw.save(raw_file_bids, overwrite=overwrite)
+        else:
+            if os.path.exists(raw_file_bids):
+                if overwrite:
+                    os.remove(raw_file_bids)
+                    sh.copyfile(raw_file, raw_file_bids)
+                else:
+                    raise ValueError('"%s" already exists. Please set'
+                                     'overwrite to True.' % raw_file_bids)
+
```

Comment:
Note that the saving procedure for MEG is as it was before this PR. there might be an error that somehow passed through the tests though: Checking the if-clauses, you save if:
- it's `.fif` or `.gz`
- if its any other MEG format AND the path exists AND `overwrite=True`

--> however, what if the format is NOT `.fif`/`.gz` and the path does not exist?

Reply:
indeed, can you make a PR?

cc @monkeyman192

Reply:
yes, but let's wait until this one is merged? I'll have a lot of rebasing to do otherwise. For now, I will make an issue so that we don't forget

Reply:
okay sounds good